playground/vitestSetup.ts

```diff
@@ -54,6 +54,8 @@ export const serverLogs: string[] = []
 export const browserLogs: string[] = []
 export const browserErrors: Error[] = []
 
+export let resolvedConfig: ResolvedConfig = undefined!
+
```

Comment:
👀 what, could we just use `resolvedConfig?.` below?

Reply:
I'm following the same pattern Anthony used for the other exports here.